run_hector.R

```diff
@@ -0,0 +1,62 @@
+# title title
+# library(dplyr)
+
+args = commandArgs(trailingOnly=TRUE)
+
+print(args)
+fn <- paste0("out_", as.character(args[1]), "_",
```

Comment:
Note that `commandArgs` returns a character vector, so these `as.character` calls aren't needed.